glib/src/auto/functions.rs

```diff
@@ -864,50 +856,3 @@ pub fn uuid_string_is_valid(str: &str) -> bool {
 pub fn uuid_string_random() -> crate::GString {
     unsafe { from_glib_full(ffi::g_uuid_string_random()) }
 }
-
-#[cfg(windows)]
-#[cfg_attr(docsrs, doc(cfg(windows)))]
-#[doc(alias = "g_win32_check_windows_version")]
-pub fn win32_check_windows_version(
```

Comment:
This and below also changed names (`glib::win32_blablabla` to `glib::win32::blablabla`)